city_scrapers/spiders/chi_ssa_21.py

```diff
@@ -0,0 +1,161 @@
+# -*- coding: utf-8 -*-
+from city_scrapers.spider import Spider
+from city_scrapers.constants import COMMISSION
+import dateutil.parser
+
+
+class ChiSsa21Spider(Spider):
+
+    name = 'chi_ssa_21'
+    agency_name = 'Lincoln Square Neighborhood Improvement Program'
+    timezone = 'America/Chicago'
+    allowed_domains = ['www.lincolnsquare.org']
+    start_urls = ['http://www.lincolnsquare.org/SSA-no-21-Commission-meetings']
+
+    def parse(self, response):
+        """
+        `parse` should always `yield` a dict that follows the Event Schema
+        <https://city-bureau.github.io/city-scrapers/06_event_schema.html>.
+
+        Change the `_parse_id`, `_parse_name`, etc methods to fit your scraping
+        needs.
+        """
+        for item in response.xpath('//div[@id="content-327081"]/p'):
+
+            data = {
+                '_type': 'event',
+                'name': self._parse_name(item),
+                'event_description': self._parse_description(item),
+                'classification': self._parse_classification(item),
+                'start': self._parse_start(item),
+                'end': self._parse_end(item),
+                'all_day': self._parse_all_day(item),
+                'location': self._parse_location(item),
+                'documents': self._parse_documents(item),
+                'sources': self._parse_sources(item),
+            }
+
+            data['status'] = self._generate_status(data, text='')
+            data['id'] = self._generate_id(data)
+
+            yield data
+
+    def _parse_name(self, item):
+        """
+        Parse or generate event name.
+        """
+        return 'Commission Meetings'
+
+    def _parse_description(self, item):
+        """
+        Parse or generate event description.
+        """
+        description = 'N/A'
+
+        # The itinerary of the meeting is always stored in the <ul>
+        # element immediately following
+        detailElement = item.xpath('following-sibling::*[1]')
+        name = detailElement.xpath('name()').extract_first()
+
+        if (name == 'ul'):
```

Comment:
The parsing here is great! The output is a little confusing though. If we could split each segment with newlines and then the topic and detail with a colon so that:

"2019 Budget Adjustments[Review and Approval]:PaintWorks[Artist Selection]"

becomes:

2019 Budget Adjustments: Review and Approval
PaintWorks: Artist Selection

that would be ideal

Reply:
Thanks for making those changes! Just two more small items. If you could default the description to an empty string instead of "N/A" and include a space after the colon so that "2019 Budget Adjustments:Review and Approval" becomes "2019 Budget Adjustments: Review and Approval" this looks good